lib/Core/EpisodeAction/EpisodeActionData.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
+
+namespace OCA\GPodderSync\Core\EpisodeAction;
+
+use DateTime;
+use JsonSerializable;
+use SimpleXMLElement;
+
+class EpisodeActionData implements JsonSerializable {
+	private ?string $podcastUrl;
+	private ?string $episodeUrl;
+	private ?string $action;
+	private int $position;
+	private int $started;
+	private int $total;
+	private int $timestampEpoch;
+
+	public function __construct(
+		?string $podcastUrl,
+		?string $episodeUrl,
+		?string $action,
+		int $position = 0,
+		int $started = 0,
+		int $total = 0
+	) {
+		$this->podcastUrl = $podcastUrl;
+		$this->episodeUrl = $episodeUrl;
+		$this->action = $action;
+		$this->position = $position;
+		$this->started = $started;
+		$this->total = $total;
+	}
+
+	/**
+	 * @return string|null
```

Comment:
Please drop all the phpdoc (in all files of this PR) unless it specifies exceptions or array element type